zipline/finance/performance/tracker.py

```diff
@@ -296,6 +296,8 @@ def process_trade(self, event):
                 perf_period.handle_cash_payment(cash_adjustment)
 
     def process_transaction(self, event):
+        # Transaction events also count as trade events for the trackers
+        self.process_trade(event)
```

Comment:
Also, this is another place where lazy-mainline handles the behavior in a different place. (Where process_trade is no longer called, the cash adjustment is done directly in the tracker.)